src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java

```diff
@@ -160,14 +162,12 @@ public void setEatingHaystack(boolean eatingHaystack)
 	}
 
 	@Override
-	@Deprecated
```

Comment:
Was this mistakenly marked as deprecated or is this a mistake?

Reply:
Those were mistakenly marked as deprecated